Jenkinsfile

```diff
@@ -1 +1,71 @@
-buildPlugin(jenkinsVersions: [null, '2.32.3'])
+// The plugin must be built on a docker platform.
+buildPlugin(platforms: ['docker'])
+
+
+node('windows') {
+    timeout(60) {
+        stage ('Test (windows-8)') {
+
+            dir('durable-task') {
+                deleteDir()
+                checkout scm
+                bat 'mkdir target\\classes'
+                bat 'mkdir target\\hpi'
+                // Need compiled java jar. Since we don't know the specific name, and because
+                // multiple jars are archived, get the hpi that contains the compiled jar
+                unarchive mapping: ['**/*.hpi': 'hpi']
```

Comment:
That's definitely a lot simpler and, like you said, would address @jglick 's concerns about  accidentally building/releasing without the go binaries.  
Another +1 in that it would allow us to also test Linux and WIndows in parallel again.

Reply:
@dwnusbaum I was going through the [`buildPlugin` code](https://github.com/jenkins-infra/pipeline-library/blob/66a14842e350bd5364a1a70ca4287535cacc0390/vars/buildPlugin.groovy) and I don't think it allows properties to be directly passed. I think it only takes in a defined list of flags.

Reply:
was able to use `withEnv` to  pass environment variable to maven.  `Jenkinsfile` can now specifically turn off binary generation for  the `windows` environment.  The `pom` will, by default, always attempt to generate the  binaries.